### ISSUE TITLE
Traffic Signs Recognition Project based on CNN

### ISSUE THREAD

**Issue by vaishnavi-1** [Jayshah6699/datascience-mashup#44]
I can work on this issue. @Jayshah6699 assign the labels to this issue and then assign it to me.

**Comment by Jayshah6699** [comment]
I am gonna need the dataset you are using and your approach towards the solution. @vaishnavi-1

**Comment by vaishnavi-1** [comment]
@Jayshah6699 I will use the dataset from kaggle and then train that by making a CNN model which will use keras, tensorflow, matplotlib and numpy. In this model, I will build a deep neural network model that can classify traffic signs present in the image into different categories.

**Comment by vaishnavi-1** [comment]
@Jayshah6699 I am confused I should do this or not..as I have seen that there is a similar issue #9 now only, and that is being done by some other person. Should I add any other project?

**Comment by Jayshah6699** [comment]
@vaishnavi-1 it's fine if you have not copied from somewhere then his work will be different than you and will remain in one folder.
